src/components/tables/DataExplorerTable/index.jsx

```diff
@@ -22,11 +138,21 @@ class DataExplorerTable extends React.Component {
 DataExplorerTable.propTypes = {
   arrangerConfig: PropTypes.object,
   arrangerData: PropTypes.object,
+  selectedTableRows: PropTypes.array,
+  api: PropTypes.func,
+  projectId: PropTypes.string,
+  setSelectedTableRows: PropTypes.func,
+  graphqlField: PropTypes.string,
```

Comment:
What is going on here?
`projectId` is always `search` in the UX
`api` points at the api fetch helper ?
`graphqlField` ?

Reply:
Here is using `api` to ask arranger for file index...and it is asking `projectId` as parameters, I don't see any other arranger API we can use to fetch data...
`graphqlField` is for the text describing how many `Subject` selected.

Reply:
The text that appears from Arranger will default to the `graphqlField` that is passed to it, so we get that for free. Another reason I would push for not changing the text

Reply:
Yes thanks for explaining! I also got uncomfortable to use this `graphqlField` but that's the only props we can get name from `araranger`...Again it would be easier to not change the text and avoid this hacking code 😛